notifications/api.py

```diff
@@ -98,5 +101,8 @@ def send_email_notification_batch(notification_ids):
         notification_ids (list of int): notification ids to send
     """
     for notification in EmailNotification.objects.filter(id__in=notification_ids):
-        notifier = _get_notifier_for_notification(notification)
-        notifier.send_notification(notification)
+        try:
+            notifier = _get_notifier_for_notification(notification)
+            notifier.send_notification(notification)
+        except:  # pylint: disable=bare-except
+            log.exception('Error sending notification')
```

Comment:
Should any exceptions that might occur here trigger a Sentry alert?  If so maybe these exceptions could be tracked over the iterations of the loop, and when the loop is complete raise an exception if errors do occur.